iib/workers/tasks/build.py

```diff
@@ -840,6 +896,40 @@ def handle_add_request(
             (prebuild_info['distribution_scope'] in ['dev', 'stage']),
         )
 
+        # Add the max ocp version property
+        # We need to ensure that any bundle which has deprecated/removed API(s) in 1.22/ocp 4.9
+        # will have this property to prevent users from upgrading clusters to 4.9 before upgrading
+        # the operator installed to a version that is compatible with 4.9
+
+        # Get the CSV name and version (not just the bundle path)
+        db_path = temp_dir + "/index.db"
+        port, rpc_proc = _serve_index_registry(db_path)
+
+        rawBundles = run_cmd(
+            ['grpcurl', '-plaintext', f'localhost:{port}', 'api.Registry/ListBundles'],
+            exc_msg='Failed to get bundle data from index image',
+        )
+        rpc_proc.kill()
+
+        # Transform returned data to parsable json
+        updated_present_bundles = [
+            json.loads(bundle) for bundle in re.split(r'(?<=})\n(?={)', rawBundles)
+        ]
+        updated_bundles = list(
+            filter(lambda b: b['bundlePath'] in resolved_bundles, updated_present_bundles)
+        )
+
+        for bundle in updated_bundles:
+            if _requires_max_ocp_version(bundle['bundlePath']):
+                maxOpenShiftVersionProperty = {
```

Comment:
The convention for Python variables are underscores, not camel case

Reply:
ah good to know! coming from golang :)